### ISSUE TITLE
Artifacts from app stays on screen after quiting nvim with nvim-tree.

### ISSUE THREAD

**Issue by pilarskikt** [vercel/hyper#7352]
- [x ] I am on the [latest](https://github.com/vercel/hyper/releases/latest) Hyper.app version
- [ ] I have searched the [issues](https://github.com/vercel/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: MacOS Ventura 13.4.1 (22F82)
- **Hyper.app version**: 3.4.1
- **Link of a [Gist](https://gist.github.com/) with the contents of your hyper.json**: https://gist.github.com/pilarskikt/06c32bdbe9e45894957cf3eee4cf8790
- **Relevant information from devtools** _(CMD+ALT+I on macOS, CTRL+SHIFT+I elsewhere)_: N/A
- **The issue is reproducible in vanilla Hyper.app**: applicable

## Issue
After fresh install of Hyper, when i enter lunarVim open nvim-tree and then exit lvim completly there are some vertical lines/artifacts from lvim session on screen, please see screenshot below.
<img width="619" alt="image" src="https://github.com/vercel/hyper/assets/18257317/391d9b9c-7af0-46d1-a604-9d649e7c1072">

**Comment by LabhanshAgrawal** [comment]
Yeah, the xterm webgl renderer sometimes has these artifacts, especially with ligatures, you can try to disable either webgl or ligatures and see if it's better.

**Comment by SamHep0803** [comment]
Having the same issue, disabling webgl and ligatures did not help in this case.

**Comment by onionka** [comment]
Can confirm, this was culprit of my errors for long time, breaking everything (VIM, TMUX, ...). I was so happy about WebGL being fast that I missed how it renders text (it draws fast, but lefts artifacts and drawing errors ... kind of reminds me prehistoric Intel processors - computes results faster than anybody on earth, but with errors, so it is useless).
